src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/FulfilmentsServiceImplTest.java

```diff
@@ -79,6 +79,7 @@ public void testFulfilmentsQuery() throws Exception {
     assertNull(capturedExampleProduct.getFulfilmentCode());
     assertNull(capturedExampleProduct.getDescription());
     assertNull(capturedExampleProduct.getHandler());
+    assertNull(capturedExampleProduct.getLanguage());
```

Comment:
This is a poor test. Can we at least set the product to have a valid language, and assert that it is the correct one here.

Reply:
As per the comment in the code block 
` // Verify that nothing else was specified in the product search`
We are checking nothing else is set in  the product serach.

Reply:
OK fair enough, just needed to add language in the mock , so its a better test.